### PR TITLE
feat: integrate cloud event for webhook

### PR DESCRIPTION
**Pull request by chlins**
Link: https://github.com/goharbor/harbor/issues/18209

Signed-off-by: chlins <chenyuzh@vmware.com>

### CODE REVIEW

**Comment by chlins** [comment]
@goharbor/all-maintainers Please help to review this proposal, thanks!

**Review by Vad1mo — CHANGES_REQUESTED** [review]
The pyload format name should be correctly set to "CloudEvents"

File: proposals/new/cloudevent.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
+# Proposal: Support Cloud Event For Webhook
```
[the remark this answers is not in the document]
Comment (chlins) [reply proposals/new/cloudevent.md]: Thanks, updated.

**Review by wy65701436 — APPROVED** [review]
lgtm

**Review by AllForNothing — APPROVED** [review]
lgtm

**Review by stonezdj — APPROVED** [review]

**Comment by chlins** [comment]
@Vad1mo Hi, the comments have been resolved, please help to review again, thanks.

**Review by Vad1mo — APPROVED** [review]
🎉

**Comment by OrlinVasilev** [comment]
That's fantastic folks!!!!!!!!

**Pull request merged**